src/git/zcl_abapgit_git_transport.clas.testclasses.abap

```diff
@@ -99,4 +100,25 @@ CLASS ltcl_git_transport IMPLEMENTATION.
 
   ENDMETHOD.
 
+  METHOD gitlab_flush_all_ok.
+
+    " GitLab and Bitbucket use 0000 opposed to GitHub which uses 00000000
+
+    DATA lv_status TYPE string.
+    DATA lx_error TYPE REF TO zcx_abapgit_exception.
+
+    lv_status = `000eunpack ok` &&
+      zif_abapgit_definitions=>c_newline &&
+      `0019ok refs/heads/master` &&
+      zif_abapgit_definitions=>c_newline &&
+      `0000`.
+
+    TRY.
+        zcl_abapgit_git_transport=>check_report_status( lv_status ).
+      CATCH zcx_abapgit_exception.
+        cl_abap_unit_assert=>fail( ).
+    ENDTRY.
```

Comment:
instead of this part, just set the method to `RAISING zcx_abapgit_exception`, its less code